src/VisualStudio/Core/Def/DocumentOutline/SortOption.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.VisualStudio.LanguageServices.DocumentOutline
+{
+    /// <summary>
+    /// The Sort Option to be applied to the document symbol data model.
+    /// </summary>
+    internal enum SortOption
+    {
+        // Sort by document symbol name.
+        Name,
+        // Sort by document symbol location in a document (by comparing each symbol's range start position).
+        Location,
+        // Sort by document symbol Microsoft.VisualStudio.LanguageServer.Protocol.SymbolKind.
+        // At the moment, we order the symbols by the SymbolKind enum values.
```

Comment:
nit: "If this ordering turns out to be undesirable, we can always add a preferred ordering later on".